tests/tree_util_tests.py

```diff
@@ -70,6 +70,15 @@ def testRoundtrip(self, inputs):
     actual = tree_util.tree_unflatten(tree, xs)
     self.assertEqual(actual, inputs)
 
+  @parameterized.parameters(*PYTREES)
```

Comment:
Maybe add an empty PyTree to PYTREES? (i.e. no leaves, and maybe a containerless leaf too)